### ISSUE TITLE
Errors in Russian - Creating Partners

### ISSUE THREAD

**Issue by Max13** [frapi/frapi#162]
Hello !

I've set my Frapi in French, and when creating a partner, the error (no company) and the confirmation are in russian :/

http://i48.tinypic.com/dqrryr.png

**Comment by davidcoallier** [comment]
This is really strange. I'll take a look.

**Comment by dshafik** [comment]
@davidcoallier Do you have time to get to this in the next 2 days for the release? If not, I'll push it off to 0.1.2 (or 0.2.0)

**Comment by davidcoallier** [comment]
Yeah will get it done. I can't replicate but I'll add the missing translations at least.

**Comment by dshafik** [comment]
Moved to 0.1.2 milestone

**Comment by trevormorse** [comment]
@Max13 Could you please try this again when you have a moment with the latest version of frapi? I've just tested in the latest version and although we are missing translations I do not see Russian...

This is what I see now: http://oi49.tinypic.com/zo88qc.jpg
